src/cascadia/TerminalCore/TerminalDispatch.cpp

```diff
@@ -394,6 +394,18 @@ bool TerminalDispatch::ResetMode(const DispatchTypes::ModeParams param) noexcept
     return _ModeParamsHelper(param, false);
 }
 
+// Method Description:
+// - Updates the current working directory.
+// Arguments:
+// - hostname: The hostname
+// - path: The path. Note that the path may not be valid on Windows.
+// Return Value:
+// - true
+bool TerminalDispatch::SetWorkingDirectory(const std::wstring_view /*hostname*/, const std::wstring_view /*path*/) noexcept
+{
+    return true;

```

Comment:
Should this return false when it's not implemented yet? maybe? (does it matter - probably no)

Reply:
Does not really matter at this moment. Will need to modify this after the profile-related things.